src/Form/Type/SimpleFormatterType.php

```diff
@@ -33,7 +33,10 @@ public function __construct(
     public function buildView(FormView $view, FormInterface $form, array $options): void
     {
         $defaultConfig = $this->ckEditorConfiguration->getDefaultConfig();
-        \assert(null !== $defaultConfig);
```

Comment:
This is not true, unless you configure on FOSCKEditorBundle a default configuration, this returns null: 

https://symfony.com/bundles/FOSCKEditorBundle/current/usage/config.html#define-a-configuration

```
The first configuration defined will be used as default configuration if you don't explicitly configure it.
```